extraTs/patterns/diamond.ts

```diff
@@ -0,0 +1,31 @@
+export function createDiamondShape(rows) {
```

Comment:
Add argument type and return type to function

Reply:
ok sir